Test/civl/assert-not-first-4.bpl

```diff
@@ -0,0 +1,17 @@
+// RUN: %parallel-boogie "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+procedure {:yields} {:layer 1} P(arr: [int]int, i: int)
+requires {:layer 1} arr[i] == 1;
+{
+    call foo(arr, i);
```

Comment:
Adding a comment that says that this is a failing example would be helpful. I spent a minute trying to understand it, before I realized that the `.expect` files says that it is supposed to fail :)